test/FluentMigrator.Tests/appsettings.json

```diff
@@ -17,8 +17,9 @@
             "IsEnabled": false
         },
         "SqlServer2016": {
-            "ConnectionString": "server=.\\MSSQLSERVER2016;Trusted_Connection=yes;database=FluentMigrator",
-            "IsEnabled": false
+            "ConnectionString": "server=.,1433;User Id=sa;Password=yourStrong(!)Password;database=master;Encrypt=false",
```

Comment:
is the sql server container using LocalDB or Developer Edition?

Reply:
It is 2019 Developer Edition from this image : 

mcr.microsoft.com/mssql/server:2019-CU18-ubuntu-20.04 (https://hub.docker.com/r/microsoft/mssql-server)

![image](https://github.com/user-attachments/assets/e0bce5c5-2543-47e8-8588-e9e42f2db03d)